### DISCUSSION TITLE
remap() edge case...

### DISCUSSION THREAD

**Discussion by villares** [py5coding/py5generator#108] [Q&A] [answered]
In the old Processing IDE some edge cases of `map()` return `NaN`/ `nan` 

![image](https://user-images.githubusercontent.com/3694604/183001037-a6a1a56e-48d2-458c-84e0-64a2c43ec6f5.png)

Now, py5 raises a `ZeroDivisionError: float division by zero`, which seems fair also.

Should we document this somehow or print a warning? Or chance it to be more like Processing?

I really don't know!

**Comment by hx2A** [comment, marked as answer]
This is bug. It shouldn't throw an error. I can change it to return `NaN`. Should it print a warning also? Probably, the warning is useful because it will help debugging efforts.
